docs/0002-N-tuple-tree.md

```diff
@@ -0,0 +1,180 @@
+                    ├── d45be626e036
+                    |   ├── 0=ocfl_object_1.0
+                    |   ├── inventory.json
+                    |   ├── inventory.json.sha512
+                    |   └── v1...
+                    ├── 3104edf0363a
+                    |   ├── 0=ocfl_object_1.0
+                    |   ├── inventory.json
+                    |   ├── inventory.json.sha512
+                    |   └── v1...
+                    └── ...
+                
+
+* PairTree: [PairTree] is designed to overcome the limitations on the number of files in a directory that most file systems have. It creates hierarchy of directories by mapping identifier strings to directory paths two characters at a time. For numerical identifiers specified in hexadecimal this means that there are a maximum of 256 items in any directory which is well within the capacity of any modern filesystem. However, for long identifiers, pairtree creates a large number of directories which will be sparsely populated unless the number of objects is very large. Traversing all these directories during validation or rebuilding operations can be slow.
+  * identifer length = 12
+  * case mapping = ToLower
+  * invert mapping = FALSE
+  * tuple size = 2
+  * number of tuples = 6
+  * short object root = FALSE
+
+                [storage_root]
+                    ├── 0=ocfl_1.0
+                    ├── ocfl_1.0.html (optional copy of the OCFL specification)
+                    ├── d4
+                    |   └── 5b
+                    |       └── e6
+                    |           └── 26
+                    |               └── e0
+                    |                   ├── 24
+                    |                   |   └──d45be626e024
+                    |                   |       ├── 0=ocfl_object_1.0
+                    |                   |       └── ...
+                    |                   └── 36
+                    |                       └──d45be626e036
+                    |                           ├── 0=ocfl_object_1.0
+                    |                           └── ...
+                    ├── 31
+                    |   └── 04
+                    |       └── ed
+                    |           └── f0
+                    |               └── 36
+                    |                   └── 3a
+                    |                       └── 3104edf0363a
+                    |                           ├── 0=ocfl_object_1.0
+                    |                           └── ...
+                    └── ...
+                
+
+* Truncated n-tuple Tree: This approach aims to achieve some of the scalability benefits of PairTree whilst limiting the depth of the resulting directory hierarchy. To achieve this, the source identifier can be split at a higher level of granularity, and only a limited number of the identifier digits are used to generate directory paths. For example, using triples and three levels with example above yields:
```

Comment:
typo: "...three levels with [the] example above yields:"

Reply:
Fixed in new PR